services/quiz.js

```diff
@@ -1,22 +1,37 @@
-const Quiz = require('~/models/quiz')
-
-const quizService = {
-  getQuiz: async (match, sort, skip = 0, limit = 10) => {
-    const items = await Quiz.find(match).skip(skip).limit(limit).sort(sort).lean().exec()
-    const count = await Quiz.countDocuments(match)
-
-    return { items, count }
-  },
-
-  createQuiz: async (author, data) => {
-    const { title, items } = data
-
-    return await Quiz.create({
-      title,
-      author,
-      items
-    })
-  }
-}
-
-module.exports = quizService
+const Quiz = require('~/models/quiz')
+const { createForbiddenError } = require('~/utils/errorsHelper')
+
+const quizService = {
+  getQuiz: async (match, sort, skip = 0, limit = 10) => {
+    const items = await Quiz.find(match).skip(skip).limit(limit).sort(sort).lean().exec()
+    const count = await Quiz.countDocuments(match)
+
+    return { items, count }
+  },
+
+  createQuiz: async (author, data) => {
+    const { title, items } = data
+
+    return await Quiz.create({
+      title,
+      author,
+      items
+    })
+  },
+
+  deleteQuiz: async (id, currentUser) => {
+    const { id: currentUserId } = currentUser
```

Comment:
you can destructure userId in controller and pass it service

Reply:
Moved.